crates/bevy_math/src/direction.rs

```diff
@@ -655,6 +690,15 @@ impl Dir3A {
         );
         Dir3A(quat.mul_vec3a(self.0))
     }
+
+    /// Returns `self` after an approximate normalization, assuming the value is already nearly normalized.
+    /// Useful for preventing numerical error accumulation.
+    #[inline]
+    pub fn fast_renormalize(self) -> Self {
+        let length_squared = self.0.length_squared();
+        // Based on a taylor approximation of the inverse square root, see `[Dir3::fast_renormalize]` for more details.
```

Comment:
```suggestion
        // Based on a taylor approximation of the inverse square root, see [`Dir3::fast_renormalize`] for more details.
```

Ditto, please see above comment :)